R/metadata.R

```diff
@@ -52,6 +52,7 @@
 #' }
 #'
 metadata <- function(surveyID,
+                     survey_definition,
```

Comment:
Let's make this an argument called "endpoint" with two options for now that match the names of the API endpoints we're using. Let's use [`rlang::arg_match()` to match them](https://rlang.r-lib.org/reference/arg_match.html), and the endpoint we're using right now ("surveys") as the default, i.e. first.

Reply:
This argument will also [need to be documented](https://r-pkgs.org/man.html).